x/init.go

```diff
@@ -64,3 +67,13 @@ To say hi to the community       , visit https://dgraph.slack.com.
 		os.Exit(0)
 	}
 }
+
+// Printf does a log.Printf. We often do printf for debugging but has to keep
+// adding import "fmt" or "log" and removing them after we are done.
+// Let's add Printf to "x" and include "x" almost everywhere. Caution: Do remember
+// to call x.Init. For tests, you need a TestMain that calls x.Init.
+func Printf(format string, args ...interface{}) {
```

Comment:
Or, you could just use your debugflag and make all the traces do a log.Printf(..), which is what I did recently. That way, you'd only add traces always, even while debugging -- which is what we should be using. Don't have any strong feelings about this, so just a thought. Feel free to ignore.